docs/user/technical-reference/05-50-ory-limitations.md

```diff
@@ -1,37 +1,42 @@
 
 The default configuration for larger clusters includes the following settings for the Ory components' resources:
 
-| Component          |          | CPU         | Memory |
-|--------------------|----------|-------------|--------|
-| Oathkeeper         | Limits   | 10 (10000m) | 512Mi  |
-| Oathkeeper         | Requests | 100m        | 64Mi   |
-| Oathkeeper Maester | Limits   | 400m        | 1Gi    |
-| Oathkeeper Maester | Requests | 10m         | 32Mi   |
+| Component          | CPU Requests | CPU Limits | Memory Requests | Memory Limits |
+|--------------------|--------------|------------|-----------------|---------------|
+| Oathkeeper         | 100m         | 10000m     | 64Mi            | 512Mi         |
+| Oathkeeper Maester | 10m          | 400m       | 32Mi            | 1Gi           |
 
 The default configuration for smaller clusters includes the following settings for the Ory components' resources:
 
-| Component          |          | CPU  | Memory |
-|--------------------|----------|------|--------|
-| Oathkeeper         | Limits   | 100m | 128Mi  |
-| Oathkeeper         | Requests | 10m  | 64Mi   |
-| Oathkeeper Maester | Limits   | 100m | 50Mi   |
-| Oathkeeper Maester | Requests | 10m  | 20Mi   |
+| Component          | CPU Requests | CPU Limits | Memory Requests | Memory Limits |
+|--------------------|--------------|------------|-----------------|---------------|
+| Oathkeeper         | 10m          | 100m       | 64Mi            | 128Mi         |
+| Oathkeeper Maester | 10m          | 100m       | 20Mi            | 50Mi          |
 
 
 ## Autoscaling Configuration
 
 The default configuration in terms of autoscaling of Ory components is as follows:
 
-| Component          | Min replicas       | Max replicas       |
-|--------------------|--------------------|--------------------|
-| Oathkeeper         | 3                  | 10                 |
-| Oathkeeper Maester | Same as Oathkeeper | Same as Oathkeeper |
-
-Oathkeeper Maester is set up as a separate container in the same Pod as Oathkeeper. Because of that, their autoscaling configuration is similar.
-
-The autoscaling is based on CPU utilization, with HorizontalPodAutoscaler set up for 80% CPU request average utilization.
+| Component          | Min replicas | Max replicas |
+|--------------------|--------------|--------------|
+| Oathkeeper         | 3            | 10           |
+| Oathkeeper Maester | 3            | 10           |
 
+Oathkeeper Maester is a separate container running in the same Pod as Oathkeeper. Because of that, autoscaling configuration of Oathkeeper and Oathkeeper Master components is similar. The autoscaling configuration is based on CPU utilization, with HorizontalPodAutoscaler set up for 80% average CPU request utilization.
```

Comment:
```suggestion
Oathkeeper Maester is a separate container running in the same Pod as Oathkeeper. Because of that, the autoscaling configuration of the Oathkeeper and Oathkeeper Master components is similar. The autoscaling configuration is based on CPU utilization, with HorizontalPodAutoscaler set up for 80% average CPU request utilization.
```